cmake/modules/boards.cmake

```
@@ -131,7 +131,7 @@ foreach(root ${BOARD_ROOT})
     message(WARNING "BOARD_ROOT element without a 'boards' subdirectory:
 ${root}
 Hints:
-  - if your board directory is '/foo/bar/boards/<ARCH>/my_board' then add '/foo/bar' to BOARD_ROOT, not the entire board directory
+  - if your board directory is '/foo/bar/boards' then add '/foo/bar' to BOARD_ROOT, not the entire board directory
```

it's not recommended to place your board directly in the `boards` folder, so please keep the `my_boards` and just remove the `<ARCH>` part.

```suggestion
  - if your board directory is '/foo/bar/boards/my_board' then add '/foo/bar' to BOARD_ROOT, not the entire board directory
```